### ISSUE TITLE
is aarch64 architecture binary cache considered to be mirrored?

### ISSUE THREAD

**Issue by AleXoundOS** [dramforever/nix-sync#1]
According to nixos.org Raspberry Pi 3 is officially supported, hydra builds 18.09 channel and uploads binaries to cache.nixos.org. This is aarch64 architecture.

I couldn't find `store-paths.xz` for aarch64. Anyway, it's pretty useless, we know, since it's not a closure.

Mirror for aarch64 is especially useful, because of a much slower builds.

**Comment by dramforever** [comment]
> I couldn't find store-paths.xz for aarch64.

That's exactly the problem.

> it's not a closure.

We can make one by 'crawling' cache.nixos.org. So the problem is that we don't know the starting points.

By the way, this repo is basically at a concept level, and no actual mirroring is done (yet). Since Nix 2.0 supports store abstractions, even the script I wrote is obsolete.

**Comment by AleXoundOS** [comment]
> By the way, this repo is basically at a concept level, and no actual mirroring is done (yet). Since Nix 2.0 supports store abstractions, even the script I wrote is obsolete.

Yeah, I see. Recently got to this problem, when I have Raspberry Pi 3 in a location without internet. After 2 weeks I found your project by following IRC logs.
